src/EnlistmentInfo.targets

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@AArnott the idea behind CakeBuild would be that it encapsulates your entire build, from compilation to deployment, without tying you to a particular IDE, or CI server, as well as performing the build cross platform.  It uses C# scripting to orchestrate the build, which is immediately more familiar to people as they are already coding the application itself in C#.